docs/pages/versions/unversioned/guides/adhoc-builds.md

```diff
@@ -157,6 +160,55 @@ An invalid profile can be caused by revoking the distribution certificate or dis
 
 ![Invalid Profile](/static/images/adhoc-builds-invalid-profile.png)
 
+# Frequently Asked Questions
+
+## Distribution certificate CLI options
+
+`expo-cli` will give you options to produce a distribution certificate, and this section describes these options in more detail.
+
+### Creating a new certificate (recommended)
+
+This will create a new iOS distribution certificate with a password. This option won't be available if you've already reached Apple's limits of 2 active certificates in your development account.
```

Comment:
```suggestion
This will create a new iOS distribution certificate with a password. This option won't be available if you've already reached Apple's limit of two active certificates in your developer account.
```